scripts/find-updates.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -euo pipefail
+# set -x
+
+# referenced from https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+function get_latest_release() {
+  version=$(curl --silent "https://api.github.com/repos/$1/releases/latest" |
+    grep '"tag_name":' |
+    sed -E 's/.*"([^"]+)".*/\1/')
+}
+
+workdir="${HOME}/work/lokomotive"
+format="%-20s %18s %18s\n"
+printf "${format}" "Component" "Current Version" "Latest Version"
+printf "${format}" "---------" "---------------" "--------------"
+
+###########################
+# k8s
+current_version=$(grep 'k8s.gcr.io/kube-apiserver' assets/lokomotive-kubernetes/bootkube/variables.tf | cut -d":" -f2)
+
+get_latest_release kubernetes/kubernetes
+printf "${format}" "kubernetes" "${current_version}" "${version}"
+
+###########################
+# calico
+current_version=$(grep 'calico/node' assets/lokomotive-kubernetes/bootkube/variables.tf | cut -d":" -f2)
+
+get_latest_release projectcalico/calico
+printf "${format}" "calico" "${current_version}" "${version}"
+
+###########################
+# etcd
+current_version=$(grep 'ETCD_IMAGE_TAG=' assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl | cut -d"=" -f3)
+
+get_latest_release etcd-io/etcd
+printf "${format}" "etcd" "${current_version}" "${version}"
+
+###########################
+# cert-manager
+
+cd "${workdir}"
+current_version=$(grep appVersion assets/components/cert-manager/manifests/Chart.yaml | cut -d":" -f2)
+
+tmpdir="/tmp/$RANDOM"
+mkdir "${tmpdir}"
+cd "${tmpdir}"
+helm repo add jetstack https://charts.jetstack.io >/dev/null 2>&1
+helm repo update >/dev/null 2>&1
+helm fetch --untar --untardir ./ jetstack/cert-manager
+version=$(grep appVersion "${tmpdir}/cert-manager/Chart.yaml" | cut -d":" -f2)
```

Comment:
Same here, we could use `yq`, though it's not as popular as `jq`.

Reply:
Should we add `yq` as dep? Because we extract the version the current version not only from the yaml files but also from terraform and go files.

Reply:
I think grep + cut is fine here :+1: